product_vision.md

```diff
@@ -0,0 +1,65 @@
+## Product Vision
+A world where blind and vision impaired people can independently and fully interact with the Windows Operating System and popular third party applications, enabling them to contribute to society, no matter their language, location or economic status.
+Blind and vision impaired people are empowered to control their own destiny and ensure this vision, through being able to significantly contribute to the technology that makes this possible.
```

Comment:
I think that under-represented people like deaf and blind should be mentioned too.

Reply:
I agree we must consider deaf-blind users always. I welcome any ideas on how that could be worded.
I would suggest though that The vision statement should continue to target just one group. So far it is "blind and vision impaired people", which of course is a superset containing "deaf-blind". So Deaf-blind people are certainly not excluded here.
* Should we make the group more explicit and say "blind, vision impaired, and deaf-blind"? Sounds very clunky and confusing to me.
* Can we extend "language, location and economic status" to include "abilities" or "other sensory abilities"?
* Should we add a third sentence talking explicitly about ensuring that blind and vision impaired people of all abilities are included? 
* What about motor and cognitive abilities? 
The vision statement can certainly be a perfect world we strive towards, but it also should be attainable, and not so large that we could never manage it.  
How ever we improve this statement, we will also need to ensure it is backed up by the guiding principles. E.g. if we do acknowledge motor abilities, then we should add voice input to the multimodal goal. Which although is the right thing to do, we also commit the project to ensuring that voice input is actually available.
Thoughts anyone?

Reply:
> • Should we make the group more explicit and say "blind, vision impaired, and deaf-blind"? Sounds Sounds very clunky and confusing to me.

I agree: this is confusing and redundant.

> Can we extend "language, location and economic status" to include "abilities" or "other sensory abilities"?

I think we can extend "language, location and economic status", but I prefer sensory characteristics instead of abilities, since characteristic sounds more qualitative instead of something measurable in terms of numerical values. Anyway I'm not a native English speaker.

> What about motor and cognitive abilities?

I thought about this too, and even mental or psichological states, not necessarily, but also mental illness. In Spain usually it's distinguished between cognitive and psichosocial (previously named mental) disabilities. I understand that NVDA itself may not contain specific code for mental abilities, but the code ofconduct aims to decrease stress, so I think that maybe mentioned in this principles.

Reply:
I have tried to address your points in my last commit. However, I have chosen to keep with the word 'abilities', as I believe that is still the most suitable word in English.
I removed language, location and economic status from the first sentence, and moved it to its own third sentence, extending it to include sensory, physical, cognitive, and mental abilities. I also added a specific example to the multimodal section of the guiding principles, which specifically mentions deaf-blind users being able to use NVDA with Braille alone.

Reply:
For me is good. Thanks.